spec/forms/review_form_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+RSpec.describe ReviewForm, type: :model do
+  let!(:current_user) { create(:user) }
+  let!(:book) { create(:book) }
+
+  let(:initialize_review) { Review.new(params) }
+  let(:create_form) { described_class.new(initialize_review, params) }
+
+  let(:params) do
+    { title: attributes_for(:review)[:title],
+      text: attributes_for(:review)[:text],
+      rating: attributes_for(:review)[:rating],
+      user_id: current_user.id,
```

Comment:
move all to lets

Reply:
fixed

Reply:
also create_form.save to one global before